File: backend/src/main/java/com/project/dto/diary/response/DiaryResponseDTO.java
package com.project.dto.diary.response;

import lombok.Data;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Data
public class DiaryResponseDTO {

    private Long diaryId;
    private String text;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: backend_dotnet/SafeMind.Core/DTOs/Diary/Response/DiaryResponseDTO.cs
namespace SafeMind.Core.DTOs.Diary.Response
{
    public class DiaryResponseDTO
    {
        public long DiaryId { get; set; }
        public long UserId { get; set; }
        public string Text { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; }
        public DateTime UpdatedAt { get; set; }
    }
}
